atlp-rwanda/commanders-medica-backoffice
# Feature: appointments dashboard

#### What does this PR do?

This PR includes designs and implementation of the appointments dashboard page.

#### Description of Task to be completed?

- Design appointments dashboard page
- Implement requests, upcoming and cancelled appointments from supabase.
- Implement appointments approval and cancellation

#### How should this be manually tested?

Start the mobile and book a doctor you used logging in the back office. After an appointment is created refresh on the appointments page, the appointment should be in the appointment requests card. Try approving it and it should be changed to the upcoming appointments card. Try to cancel it and it should be transferred in the cancelled card.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

N/A

#### Questions:

N/A
